tokio/src/util/sharded_list.rs

```diff
@@ -0,0 +1,160 @@
+#![cfg_attr(not(feature = "full"), allow(dead_code))]
+
```

Comment:
Please remove this one.

Reply:
Do you means `#![cfg_attr(not(feature = "full"), allow(dead_code))]`, if do that, we will got dede_code error in some feature.

Reply:
Please adjust the `cfg` in `src/util/mod.rs` instead to fix that dead code warning.

Reply:
I use `cfg_rt!` in `src/util/mod.rs`, and move `#![cfg_attr(not(feature = "full"), allow(dead_code))]`.